gax-java/pom.xml

```diff
@@ -205,6 +210,20 @@
       </exclusions>
       <scope>test</scope>
     </dependency>
+    <!--    Logback  for testing -->
+    <dependency>
```

Comment:
It would be great if we don't have to add logback in unit tests. Maybe create our own SLF4J test logger, similar to the JUL [TestLogger](https://github.com/googleapis/sdk-platform-java/blob/c334a0c67acaad0d3de48100fd3b7cff78f4aef5/java-core/google-cloud-core/src/test/java/com/google/cloud/testing/LogRecorderTest.java#L74)